views/user/forms/language.hbs

```diff
@@ -5,19 +5,32 @@
         class="dropdown-item toggle-dropdown fla-{{language}}"
         data-language="{{language}}"
         data-testid="selected-language-{{language}}"
-        aria-label="{{$t 'global.language.label.selected'}} {{$t (concat 'global.language.label.' language)}}"
+        aria-label="{{$t 'global.language.label.selected'}} {{$t
+            (concat 'global.language.label.' language)
+        }}"
         role="menuitem"
-    >{{$t (concat "global.language.label." language)}}</a>
+    >
+        {{$t (concat "global.language.label." language)}}
+        <i
+            class="mdi mdi-chevron-down language-menu-expand-icon"
+            aria-hidden="true"
+        ></i>
+    </a>
     <ul id="available-languages" class="dropdown-menu" role="menu">
-        {{#each availableLanguages as | lang | }}
+        {{#each availableLanguages as |lang|}}
             {{#ifCond lang "!=" @root.userLanguage}}
```

Comment:
how about adding "!==" here, or would that change too much?

Reply:
I mean, I can, but these changes are only present due to format changes. I didn't really look at these lines